torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -64,6 +63,10 @@ std::map<at::ScalarType, ncclDataType_t> ncclDataType = {
     {at::kLong, ncclInt64},
     {at::kHalf, ncclHalf},
     {at::kBool, ncclUint8},
+    {at::kFloat8_e5m2, ncclUint8},
```

Comment:
I think this is reasonable! This also means we can get rid of the uint8 `view` when communicating fp8 format?

Reply:
Yup, did that here and all dtensor test passed: https://github.com/pytorch-labs/float8_experimental/pull/263